### PR TITLE
Bump to v2.8.0

### PR DESCRIPTION
**Pull request by Onumis**
Quite a few thing in this release :)

### CODE REVIEW

**Comment by Onumis** [comment]
Not sure if we should add a run time dependency:

```
spec.add_runtime_dependency 'ruby_parser', '>= 3.8' 
```

Since it enables support for ruby 2.3 and fixes parsing of `{ 'a': :b }` on ruby 2.2.

**Comment by troessner** [comment]
>> Not sure if we should add a run time dependency:
spec.add_runtime_dependency 'ruby_parser', '>= 3.8' 
Since it enables support for ruby 2.3 and fixes parsing of { 'a': :b } on ruby 2.2.

:+1: , let's do that!

**Comment by Onumis** [comment]
@troessner updated!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,11 @@
-# unreleased
+# 2.8.0 / 2016-02-29
 
+* [FEATURE] Add link to Reek's code smells documentation (by danielmbarlow)
+* [FEATURE] Make RubyCritic usage as Rake Task (by Timo Rößner)
```
Comment (troessner) [inline CHANGELOG.md]: Typo - should be "usable"

Comment (Onumis) [inline CHANGELOG.md]: Well spotted!

**Comment by Onumis** [comment]
published

**Pull request merged**